cypress/integration/homepage.spec.js

```diff
@@ -61,3 +61,49 @@ describe("Header", () => {
     cy.get('[data-testid="search_input"]').should("be.empty");
   });
 });
+
+describe("Section 1", () => {
+  it("Displays title & subtitle", () => {
+    // Find headers and confirm their text
+    cy.get('[data-testid="section_one"]').contains(
+      "h2",
+      "Effective, long-lasting anti-corrosion protection above and below ground"
+    );
+
+    cy.get('[data-testid="section_one"]').contains(
+      "h3",
+      "Anti-corrosion wraps, wax coatings, casing fillers, and marine pile protection systems worldwide for oil, gas, water, and chemical processing companies."
+    );
+  });
+
+  it("Displays 'Call Us' and 'Send A Message' links", () => {
+    // Find all 'Call Us' and 'Send a message' links & make sure they work
+    cy.get(".callUsLink").each(($link) => {
```

Comment:
Instead of searching by class, what about finding all a tags that contain "Call us" and having cypress check their href?

Reply:
Refactored with cy.get and .contains so that we can find a tags for our button links regardless of classname or data-id.